### PR TITLE
chore: do not login to DockerHub

### PR DESCRIPTION
**Pull request by iliapolo**
See https://github.com/aws/cdk-ops/pull/887 (Will fail until this is merged)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: scripts/ecr-proxy/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+server {
+    listen       PROXY_PORT;
```
Comment (iliapolo) [inline scripts/ecr-proxy/nginx.conf:2]:
```suggestion
    listen       @PROXY_PORT@;
```

**Review by eladb — APPROVED** [review]

**Review by eladb — CHANGES_REQUESTED** [review]
Wait

File: buildspec-pr.yaml [hunk 2]
```diff
@@ -5,11 +5,9 @@ version: 0.2
 phases:
   install:
     commands:
-      # Start docker daemon inside the container
-      - nohup /usr/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay2&
-      - timeout 15 sh -c "until docker info; do echo .; sleep 1; done"
```
Comment (eladb) [inline buildspec-pr.yaml:10]: Aren't these still needed to start the docker daemon?

**Review by eladb — CHANGES_REQUESTED** [review]

File: buildspec-pr.yaml (hunk 2 above)
Comment (iliapolo) [reply buildspec-pr.yaml:10]: @eladb We decided to move this into the ecr proxy image as well - so no. the `start.sh` scripts takes care of it.

Comment (iliapolo) [reply buildspec-pr.yaml:10]: I'll add a comment and maybe later we can think about renaming/exposing another script to make it more clear.

**Comment by iliapolo** [comment]
> ### AWS CodeBuild CI Report
> * CodeBuild project: AutoBuildProject6AEA49D1-qxepHUsryhcu
> * Commit ID: [f5b4981](https://github.com/aws/aws-cdk/commit/f5b4981fe34626a7a65653edb8db66855ff3ebbc)
> * Result: FAILED
> * [Build Logs](https://6sc75qyfl7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=18dc7dea-7667-4f8f-8147-abb27012f1d6%2Fbuild.log) (available for 30 days)
> 
> _Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)_

```console
aws-cdk/aws-lambda-nodejs: FAIL test/bundling.test.js (78.783 s)
@aws-cdk/aws-lambda-nodejs:   â— Parcel bundling
@aws-cdk/aws-lambda-nodejs:     SyntaxError: Unexpected end of JSON input
@aws-cdk/aws-lambda-nodejs:         at JSON.parse (<anonymous>)
@aws-cdk/aws-lambda-nodejs:       21 |     this.pkgPath = path.join(pkgPath, 'package.json');
@aws-cdk/aws-lambda-nodejs:       22 |     this.pkg = fs.readFileSync(this.pkgPath);
@aws-cdk/aws-lambda-nodejs:     > 23 |     this.pkgJson = JSON.parse(this.pkg.toString());
@aws-cdk/aws-lambda-nodejs:          |                         ^
@aws-cdk/aws-lambda-nodejs:       24 |   }
@aws-cdk/aws-lambda-nodejs:       25 | 
@aws-cdk/aws-lambda-nodejs:       26 |   /**
@aws-cdk/aws-lambda-nodejs:       at new PackageJsonManager (lib/package-json-manager.ts:23:25)
@aws-cdk/aws-lambda-nodejs:       at Function.parcel (lib/bundling.ts:141:32)
@aws-cdk/aws-lambda-nodejs:       at Object.<anonymous>.test (test/bundling.test.ts:31:12)
```

This sure looks like a race condition...

**Review by NetaNir — APPROVED** [review]

**Pull request merged**